### ISSUE TITLE
Document how to set up management console behind reverse proxy

### ISSUE THREAD

**Issue by djuarezg** [sourcegraph/sourcegraph#2759]
#### Question description

How should we configure an Apache proxy to work with the management console.

#### Is your question related to a problem? If so, please describe or link to another issue.

We configured a proxy in front on Sourcegraph, but after the upgrade to 3.X we have to configure access to the management console as well, which uses basic auth.

The problem is that the configuration update does not happen due to `Unexpected HTTP 401: Unauthorized `

The request does not send any cookie or basic auth as a parameter/header.

**Comment by slimsag** [comment]
Apologies for the delay here. Are you still blocked on this?

Currently, you should be able to configure the reverse proxy to allow management console access via a subdomain e.g. console.sourcegraph.example.com.

For an Apache proxy, a few quick Google searches tells me it would look like this:

```
ProxyPreserveHost On
ProxyPass / http://console.sourcegraph.example.com/
ProxyPassReverse / http://localhost:2633/
```

I have not tested the above, but that should work. Does this answer the question?

For doing this under a different path than `/` (i.e. without a subdomain), we will need a change to support that #2808. If you need that, please let me know.

**Comment by slimsag** [comment]
Changing milestone to 3.5 for @ryan-blunden to document how to do this (since he also committed to testing and merging https://github.com/sourcegraph/sourcegraph/pull/2808 ), if this sounds wrong reassign me! :)
